app/clients/sms/aws_pinpoint.py

```diff
@@ -21,13 +21,17 @@ def init_app(self, current_app, statsd_client, *args, **kwargs):
     def get_name(self):
         return self.name
 
-    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None):
+    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None, service_id=None):
         messageType = "TRANSACTIONAL"
         matched = False
         opted_out = False
         response = {}
 
-        if template_id is not None and str(template_id) in self.current_app.config["AWS_PINPOINT_SC_TEMPLATE_IDS"]:
+        use_shortcode_pool = (
+            str(template_id) in self.current_app.config["AWS_PINPOINT_SC_TEMPLATE_IDS"]
+            or str(service_id) == self.current_app.config["NOTIFY_SERVICE_ID"]
```

Comment:
Why don't we just put the notify service id in the AWS_PINPOINT_SC_TEMPLATE_IDS?

Reply:
We could but we plan to move away from this env var entirely and have the pool choice be more dynamic. The `NOTIFY_SERVICE_ID` variable already exists so this way we don't need to change `AWS_PINPOINT_SC_TEMPLATE_IDS` and can eventually remove it.